packages/flutter_hooks/lib/src/debounced.dart

```diff
@@ -0,0 +1,74 @@
+part of 'hooks.dart';
+
+/// Returns a [ValueNotifier] that updates its value with a timeout
+/// for the given input [toDebounce] and triggers widget rebuilds accordingly.
```

Comment:
Some examples and more docs would be great. For example we'd want to explain why we have a way to update the value